test/WireMock.Net.Tests/Facts/IgnoreOnContinuousIntegrationFact.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright © WireMock.Net
+using System;
+using Xunit;
+
+namespace WireMock.Net.Tests.Facts
+{
+    public sealed class IgnoreOnContinuousIntegrationFact : FactAttribute
+    {
```

Comment:
``` c#
private static readonly bool _isContinuousIntegration = bool.TryParse(Environment.GetEnvironmentVariable("TF_BUILD"), out var isTF) && isTF;
```

And the unit tests are also run on GitHub actions, so an extra check is needed --> `GITHUB_ACTIONS` equals true.